migrations/app/20171004115229_app_workload_add_staging_id.js

```diff
@@ -0,0 +1,14 @@
+exports.up = function (knex, promise) {
+  var sql = 'ALTER TABLE app.workload ADD staging_id INT;' +
```

Comment:
Main comment on this one is as per our F2F discussion - need for an ID which ties together the latest set of staging Ids. As there is no constraint on this (and can't be, because data migration) we need to ensure we don't get weird behaviour on the off-chance that duplicate staging IDs make it into this table.